### DISCUSSION TITLE
request for method 'eth_chainId'  failed [Polygon]

### DISCUSSION THREAD

**Discussion by Royal-lobster** [Web3Auth/Web3Auth#866] [Help]
I am trying to connect my app with external wallet using brave to mumbai polygon, but getting this error - 

![CleanShot 2022-11-12 at 16 40 51@2x](https://user-images.githubusercontent.com/52039218/201486504-574618c4-6469-4549-b341-b5dc1913b9ba.png)

```bash
Uncaught (in promise) 
{code: 4001, message: "Request for method 'eth_chainId' on https://rpc.ankr.com/polygon_mumbai failed"}
```

**Comment by shahbaz17** [comment]
Hey @Royal-lobster 

Have you tried using a different RPC endpoint? It looks like the RPC was down. Please try custom RPC endpoints from providers like infura, quicknode, etc.

**Reply by Royal-lobster** [reply]
Thanks ! it works after changing RPC endpoint
